src/plugins/thumbnail/lg-thumbnail.ts

```
@@ -435,7 +448,16 @@ export default class Thumbnail {
         const img = document.createElement('img');
         img.alt = alt || '';
         img.setAttribute('data-lg-item-id', index + '');
-        img.src = thumbImg;
+
+        if (this.settings.thumbnailLazyLoad) {
+            let placeholder = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsQAAA7EAZUrDhsAAAANSURBVBhXYzh8+PB/AAffA0nNPuCLAAAAAElFTkSuQmCC";
```

ESLint

[error] 'placeholder' is never reassigned. Use 'const' instead.

(prefer-const)